offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/persons/CustomerDbConnector.groovy

```diff
@@ -470,7 +566,26 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
     }
     return customers
   }
-
+  
+  /**
+   * List all available persons that are set to active in the database.
+   * @return A list of active persons
+   */
+  List<Customer> fetchAllActiveCustomers() {
```

Comment:
```suggestion
  List<Customer> fetchAllActivePersons() {
```

Reply:
Because the Customer is returned I would opt for keeping the method name as is